main.go

```diff
@@ -43,8 +43,10 @@ const (
 )
 
 type Ban struct {
```

Comment:
Is this backward compatible?

Reply:
Yes, old bans are converted to new bans automatically (they default to being indefinite) and new bans are converted to old ones.

Reply:
wdym by the new bans are converted to old ones?

Reply:
A database generated in the old format can be read by a new Devzat, and a database generated in the new format can be read by an old Devzat.

Reply:
oh awesome!